plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1656,23 +1703,46 @@ bool producer_plugin_impl::remove_expired_trxs( const fc::time_point& deadline )
    size_t num_expired_other = 0;
    size_t orig_count = _unapplied_transactions.size();
    bool exhausted = !_unapplied_transactions.clear_expired( pending_block_time, deadline,
-                  [&num_expired_persistent, &num_expired_other, pbm = _pending_block_mode,
+                  [this, &num_expired_persistent, &num_expired_other, pbm = _pending_block_mode,
                    &chain, has_producers = !_producers.empty()]( const transaction_id_type& txid, trx_enum_type trx_type ) {
+
+            auto trx = this->_unapplied_transactions.get_trx(txid);
+            fc::variant entire_trx = fc::mutable_variant_object()("transaction", trx->packed_trx()->get_transaction());
```

Comment:
Only want to do this if debug logger enabled.
I don't see a need to put this under "transaction".
Use abi_serializer.

Reply:
Fixed